metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/converter/TaxonomyConverter.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package uk.ac.ebi.ampt2d.metadata.importer.converter;
+
+import org.springframework.core.convert.converter.Converter;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.Taxonomy;
+import uk.ac.ebi.ena.browser.xml.ParentTaxonType;
+
+public class TaxonomyConverter implements Converter<ParentTaxonType, Taxonomy> {
+
+    public static final String SUBSPECIES = "subspecies";
+
+    public static final String SPECIES = "species";
+
+    public static final String GENUS = "genus";
+
+    public static final String ORDER = "order";
+
+    public static final String CLASS = "class";
+
+    public static final String SUPERCLASS = "superclass";
+
```

Comment:
Because it's quite a high rank taxonomy, and is very rarely used for filtering purposes. One of the considerations when limiting the number of fixed taxonomy ranks was that it's hard to display a tree with lots of taxonomies on the website. So we decided to go with just class, order, genus and species for now. If in the future we decide to add more ranks based on usability testing, we'll do it